pybind11/setup_helpers.py

```diff
@@ -143,7 +144,12 @@ def __init__(self, *args, **kwargs):
         if WIN:
             cflags += ["/EHsc", "/bigobj"]
         else:
-            cflags += ["-fvisibility=hidden", "-g0"]
+            cflags += ["-fvisibility=hidden"]
+            env_cflags = os.environ.get("CFLAGS", "")
+            env_cppflags = os.environ.get("CFLAGS", "")
```

Comment:
```suggestion
            env_cppflags = os.environ.get("CXXFLAGS", "")
```

Reply:
Oops, fixed.  It's CPPFLAGS though (again, per the distutils source).